WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -444,7 +444,8 @@ private extension SiteAddressViewController {
         checkSiteExistence(url: url) { [weak self] in
             guard let self = self else { return }
             // skips XMLRPC check for site discovery if needed
```

Comment:
Nit: Should we update this comment to add that we skip the check for site address login case as well?